packages/artifact/src/internal/find/retry-options.ts

```diff
@@ -0,0 +1,44 @@
+import * as core from '@actions/core'
+import {OctokitOptions} from '@octokit/core/dist-types/types'
+import {RequestRequestOptions} from '@octokit/types'
+
+export type RetryOptions = {
+  doNotRetry?: number[]
+  enabled?: boolean
+}
+
+export function getRetryOptions(
```

Comment:
Copied this from here: https://github.com/actions/github-script/blob/main/src/retry-options.ts

This warrants installing `@octokit/core`, `@octokit/plugin-request-log` and `@octokit/plugin-retry` which is why you see those new additions to `package.json`